Client/Frontend/Browser/HomePanel/FavoritesViewController.swift

```diff
@@ -104,8 +130,15 @@ class FavoritesViewController: UIViewController {
         braveShieldStatsView.frame = statsViewFrame
         
         collection.addSubview(braveShieldStatsView)
+        collection.addSubview(ddgButton)
+        
+        ddgButton.addSubview(ddgLogo)
+        ddgButton.addSubview(ddgLabel)
         
         makeConstraints()
+        
+        collection.addObserver(self, forKeyPath: #keyPath(UICollectionView.contentSize), options: [.new, .initial], context: nil)
```

Comment:
Any reason not to use the newer block-based observing?

Reply:
Na, just entirely out of habit lol. Updated!